README.md

```diff
@@ -25,6 +25,23 @@ To build the final jar:
 make build
 ```
 
+### How to add a OAuth2 app for local development
+Before starting local development, you must create an OAuth application and add clientID and clientSecret to application.yaml: 
```

Comment:
This file is in Git, so changing it may cause us to accidentally run the changed file. Let's think about how we can avoid this. But in general, it could be that for local development we just set the right values, but for projection we set different values - will this work?

Reply:
@fey Hey!

> This file is in Git, so changing it may cause us to accidentally run the changed file. Let's think about how we can avoid this
To be honest, i dont get what do you mean. If you mean that other people will have to solve conflicts while pushing new changes from their own branches because files will be a different. So they can do merge it before push. I done it before i open the pr. To avoid this I can add a custom exception while building project to be done the check and don't change a README file. Will this option work for you?

> But in general, it could be that for local development we just set the right values, but for projection we set different values - will this work?
I added 2 instructions: to local delopment and to deploy the project. A Dockerfile uses the production profile. See application-PROD.yaml

Reply:
See, usually the files that need to be changed for local development are not versioned. These are different configuration files, settings that are specific to the environment and to the developer. For example, it could be the Github settings with which the connection is made. Then let's think about it, maybe just create a Github application for authorization locally.

Reply:
@fey Hey!

I added a OAuth app for dev. Fixed README
It looks like:

<img width="1470" alt="Снимок экрана 2024-08-07 в 00 35 45" src="https://github.com/user-attachments/assets/33b44072-5eef-480a-b43f-333079f4c1b8">

<img width="1471" alt="Снимок экрана 2024-08-07 в 00 35 57" src="https://github.com/user-attachments/assets/0d7f579e-e0eb-4c98-88ed-59488d4830c4">

<img width="1471" alt="Снимок экрана 2024-08-07 в 00 37 13" src="https://github.com/user-attachments/assets/91c9d95a-78b0-4792-86bb-c30220a41f10">

<img width="1471" alt="Снимок экрана 2024-08-07 в 00 36 17" src="https://github.com/user-attachments/assets/b4ed2632-ff9a-4483-b939-4939500e7ccb">

<img width="1470" alt="Снимок экрана 2024-08-07 в 00 37 34" src="https://github.com/user-attachments/assets/07d0f5a2-b6eb-479f-88e5-a6c675964810">

<img width="1471" alt="Снимок экрана 2024-08-07 в 00 37 13" src="https://github.com/user-attachments/assets/01f92e81-1a58-4757-8cad-71ecca91e94f">